python/cuml/test/utils.py

```diff
@@ -163,3 +163,44 @@ def quality_param(*args, **kwargs):
 
 def stress_param(*args, **kwargs):
     return pytest.param(*args, **kwargs, marks=pytest.mark.stress)
+
+
+class ModuleConfig:
```

Comment:
Can we rename this to ClassEnumerator or ModuleCatalog or something like that? ModuleConfig sounds like it would be about configuration management to me.

Reply:
Agreed.